dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -230,4 +230,14 @@ def create_admin_script
     get :show, id: admin_script.name
     assert_response :success
   end
+
+  test "should have one game group if one stage" do
```

Comment:
oh I see. does it also apply to header_stats like "should have two game groups if two stages" (and does that one also apply to scripts/show)?